### PR TITLE
Twitter settings e2e

### PR DESCRIPTION
**Pull request by alex-deaconu**

### CODE REVIEW

File: web/scripts/displays/services/svc-primus.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ angular.module('risevision.displays.services')
         }
 
         function lastConnectionNew(id) {
-          return presenceData[id].lastConnectionTime;
+          return Number(presenceData[id].lastConnection);
```
Comment (alex-deaconu) [inline web/scripts/displays/services/svc-primus.js]: @rudibravo curious, do you know where this change came from and what it's for? Thanks.

Comment (rudibravo) [reply web/scripts/displays/services/svc-primus.js]: @alex-deaconu it's from this PR https://github.com/Rise-Vision/rise-vision-apps/pull/587. I think the branch feature/pro-content is older than feature/twitter-settings-e2e

Comment (alex-deaconu) [reply web/scripts/displays/services/svc-primus.js]: @rudibravo got it, thanks!

**Comment by alex-deaconu** [comment]
@rudibravo FYI this PR is ready to go. Thanks for updating the e2e tests for this 👍 

@Rise-Vision/apps merging this, will get a chance to review with the main PR (#590).

Thanks.

**Pull request merged**